tests/foreman/api/test_syncplan.py

```diff
@@ -0,0 +1,399 @@
+"""Unit tests for the ``repositories`` paths."""
+import datetime
+from ddt import ddt
+from fauxfactory import gen_string
+from robottelo import entities
+from robottelo.common.decorators import (
+    data, run_only_on, stubbed)
+from robottelo.test import APITestCase
+# (too-many-public-methods) pylint:disable=R0904
+
+
+@ddt
+class SyncPlanCreateTestCase(APITestCase):
+    """Tests specific to creating new ``Sync Plans``."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create an organization which can be re-used in tests."""
+        cls.org_id = entities.Organization().create_json()['id']
+        super(SyncPlanCreateTestCase, cls).setUpClass()
+
+    @stubbed()
+    @run_only_on('sat')
+    def test_create_disabled_sync_plan(self):
+        """@Test: Create a disabled Sync Plan.
+
+        @Assert: A disabled sync plan is created.
+
+        @Feature: Sync Plan
+
+        """
+
+    @stubbed()
+    @run_only_on('sat')
+    @data(
+        gen_string('alpha', 15),
+        gen_string('alphanumeric', 15),
+        gen_string('numeric', 15),
+        gen_string('latin1', 15),
+        gen_string('utf8', 15),
+        gen_string('html', 15),
```

Comment:
hehehe just following the same pattern that was used to a similar test done via CLI.